### PR TITLE
feat(Progress): support rtl

### PR DESCRIPTION
**Pull request by eyoue**

### CODE REVIEW

File: src/components/Progress/Progress.tsx [hunk 1]
```diff
@@ -144,7 +144,7 @@ export class Progress extends React.Component<ProgressProps> {
         const className = b('item', {theme: this.getTheme(), loading: this.props.loading});
 
         const offset = Progress.getOffset(value);
-        const style = {transform: `translateX(${offset}%)`};
+        const style = {insetInlineStart: `${offset}%`};
```
Comment (amje) [inline src/components/Progress/Progress.tsx:147]: Animating margin instead of transform will cause a reflow on update. Let's keep transform

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/Progress/Progress.tsx (hunk 1 above)
Comment (eyoue) [reply src/components/Progress/Progress.tsx]: Yes, tnx. I changed to transform using css variables

File: src/components/Progress/Progress.scss [hunk 2]
```diff
@@ -7,6 +7,14 @@ $progress-m-height: 20px;
 $progress-s-height: 10px;
 $progress-xs-height: 4px;
 
+:root {
+    --flow-direction: 1;
+}
+
+[dir='rtl'] {
+    --flow-direction: -1;
+}
```
Comment (amje) [inline src/components/Progress/Progress.scss:16]: Please you local variable here, with "private" naming:
```scss
.progress {
  --_--flow-direction: 1;
  
  [dir='rtl'] & {
    --_--flow-direction: -1;
  }
}
```

**Review by amje — CHANGES_REQUESTED** [review]

**Comment by amje** [comment]
Also Progress with stack still ignores rtl

File: src/components/Progress/Progress.scss (hunk 2 above)
Comment (eyoue) [reply src/components/Progress/Progress.scss]: Tnx, i changed it.

**Comment by eyoue** [comment]
> Also Progress with stack still ignores rtl

Ops :C
Yes, really.. Comonent use float.
Ok, i changed it for rtl. Tnx.

**Review by amje — APPROVED** [review]

**Comment by amje** [comment]
@Lunory Numbers have own semantics here, 60/100 is the correct one for both LTR and RTL

**Review by Lunory — APPROVED** [review]

**Pull request merged**